charts/gardener-extension-admission-aws/charts/runtime/templates/deployment.yaml

```diff
@@ -21,6 +21,7 @@ spec:
       labels:
 {{ include "labels" . | indent 8 }}
     spec:
+      priorityClassName: gardener-system-900  
```

Comment:
Shorty question. This component will be deployed into the host cluster and interact with the gardener api-server so it's deployed into a seed or shoot nor interacting with them. So will this priority class also available in the context where this component is deployed?

Reply:
Yes, PriorityClasses `gardener-shoot-system(600-900)` are deployed to the shoot cluster as well.

Reply:
Well, but the component is not deployed to a Shoot or a Seed. It is deployed to the cluster where the Gardener components run. So will it be ensured that the `priorityClass` you referenced is also there. At least there is probably no gardenlet which could take care.

Reply:
Ok, I misunderstood you. So, if it is not part of the Shoot System Components, Seed System Components or Shoot Control Plane Components, we should skip this deployment.